### ISSUE TITLE
Clarification on whether the FP8 scaling-law loss points are from BF16 runs

### ISSUE THREAD

**Issue by jusjinuk** [IST-DASLab/Quartet#12]
@BlackSamorez @soroush-tabesh @4PiR2 

Hello, thank you for releasing the paper and code!

I went through [`notebooks/plots.ipynb`](https://github.com/IST-DASLab/Quartet/blob/main/notebooks/plots.ipynb), and it looks like the baseline loss points are stored as `BF16:BF16`, while the plotting code labels those same points as `FP8:FP8`.

Could you clarify whether the points used to fit the `FP8:FP8` curve are BF16 training losses used as a proxy for FP8 quality, with FP8 GEMM performance used only for the speed comparison, rather than losses from actual FP8-quantized training runs?

Based on Section 2 in the paper ("Training in 8-bit formats"), this seems to be the case, but I wanted to double-check.

>  In our work, we will treat FP8 as the idealized baseline that has the quality of BF16 and the speed of raw FP8 GEMM operations.

Thank you.

**Comment by BlackSamorez** [comment]
These are indeed BF16 runs. In the paper, when reporting FP8 accuracy, what we actually report is BF16 accuracy, which is a natural upper bound on the latter. This way, when comparing with this upper bound, we compare agains the best possible FP8 algorithm automatically. By doing so, we did not have to implement or benchmark any specific FP8 training algorithms.

**Comment by jusjinuk** [comment]
Thank you for the prompt reply and clarification!
